.travis.yml

```diff
@@ -124,11 +124,12 @@ jobs:
         - MATRIX_EVAL="COMPILER=gcc && CC=gcc-6 && CXX=g++-6"
         - RUN_VALGRIND=true
         - DISABLE_CI_TESTS=true
+        - BUILD_TYPE=RelWithDebInfo
         - USE_MPI=mpich
     - <<: *daily_linux
       env:
         - MATRIX_EVAL="COMPILER=gcc && CC=gcc-6 && CXX=g++-6"
-        - RUN_COVERAGE=true
+        - BUILD_TYPE=Coverage
```

Comment:
Is this a recognized CMAKE_BUILD_TYPE, or something special applied here?

Reply:
CodeCoverage.cmake file sets cmake variables for flags to use with the Coverage build type.